docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
+    <NoWarn>1701;1702;0649</NoWarn>
```

Comment:
><NoWarn>1701;1702;0649</NoWarn> [](start = 4, length = 31)

one concern i have is that users  might not be aware of this..so they will get thrown off  when they see the warning about  "uninitialized field" ...  

an alternative is to make them properties @sfilipi  what do u think ?

Reply:
Just switched to properties. No need to disable warning anymore.

---
In reply to: [261304076](https://github.com/dotnet/machinelearning/pull/2780#discussion_r261304076) [](ancestors = 261304076)